04/index.html

```diff
@@ -1,30 +1,45 @@
 <!DOCTYPE html>
 <html lang="pl">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>devmentor.pl - HTML & CSS: RWD - #04</title>
-</head>
-<body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
-            </li>
-            <li>
-                <a href="#">oferta</a>
-                <ul>
-                    <li><a href="">strony internetowe</a></li>
-                    <li><a href="">pozycjonowanie stron www</a></li>
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <title>devmentor.pl - HTML & CSS: RWD - #04</title>
+        <link rel="stylesheet" href="./CSS/global.css" />
+        <link rel="stylesheet" href="./CSS/mobile.css" />
+        <link rel="stylesheet" href="./css/desktop.css" />
+    </head>
+    <body>
+        <header id="header" class="header">
+            <nav class="nav">
+                <label for="menu-switcher">&#9776;</label>
+                <input type="checkbox" id="menu-switcher" />
+                <ul class="nav__items">
+                    <li class="nav__item">
+                        <a class="nav__link" href="#">start</a>
+                    </li>
+                    <li class="menu__dropdown">
+                        <div class="nav__link-oferta">
```

Comment:
W zasadzie można by było się obejść bez tego `div` - tak byłoby lepiej, ale to już następnym razem :)